pallets/fixed-point/Cargo.toml

```diff
@@ -3,6 +3,22 @@ name = "fixed-point"
 version = "2.0.0-alpha.6"
 authors = ["Joshy Orndorff"]
 edition = "2018"
+description = "A pallet that demonstrates using fixed point arithmetic on Substrate"
+license = "GPL-3.0-or-later"
+homepage = "https://github.com/substrate-developer-hub/recipes"
+repository = "https://github.com/substrate-developer-hub/recipes"
+documentation = "https://github.com/substrate-developer-hub/recipes"
+#readme = ""
+
+[package.metadata.substrate]
+categories = [
+	"fixed point",
+	"tutorial",
+	"recipe",
+	"arithmetic",
+]
+# icon = "path/to/icon.png"
+compatibility_version = "2.0.0-alpha.6"
```

Comment:
What does this do? I've never seen it before!

Reply:
Also: should we go for `2.0.0-alpha.7`?

Reply:
This is metadata specifically for the substrate marketplace (https://marketplace-staging.substrate.dev/) written by @stiiifff . The marketplace scrapes crates.io for crates that work with Substrate and indexes them. It wants this data.

Yes, we will go for alpha.7 my plan for today is to merge a few outstanding PRs and then make the update. There is a lot of code, so the update sometimes takes a while.